### ISSUE TITLE
[BUG]Can not add  new translation language for zh (Chinese)

### ISSUE THREAD

**Issue by waynerv** [tiangolo/fastapi#1177]
### Describe the bug

execute `python ./scripts/docs.py new-lang zh` raise UnicodeDecodeError.

### To Reproduce

Follow the steps in documents to set up environment, then add a new language:

1. Add a new language for “zh”:

```bash
python ./scripts/docs.py new-lang zh
```

2. Raise:
```
Traceback (most recent call last):
  File "./scripts/docs.py", line 338, in <module>
    app()
  File "D:\fastapi\env\lib\site-packages\typer\main.py", line 211, in __call__
    return get_command(self)()
  File "D:\fastapi\env\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "D:\fastapi\env\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "D:\fastapi\env\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "D:\fastapi\env\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "D:\fastapi\env\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "D:\fastapi\env\lib\site-packages\typer\main.py", line 494, in wrapper
    return callback(**use_params)  # type: ignore
  File "./scripts/docs.py", line 98, in new_lang
    en_index_content = en_index_path.read_text()
  File "C:\Users\amped\AppData\Local\Programs\Python\Python38\lib\pathlib.py", line 1215, in read_text
    return f.read()
UnicodeDecodeError: 'gbk' codec can't decode byte 0xa8 in position 5557: illegal multibyte sequence
```
### Workaround
1. Add `encoding="utf-8"` param for `read_text()` and `write_text()` in scripts/docs.py line 98,100 can fix this problem.
```python
    new_config["extra_javascript"] = extra_js
    new_config_path: Path = Path(new_path) / mkdocs_name
    new_config_path.write_text(yaml.dump(new_config, sort_keys=False, width=200))
    new_config_docs_path: Path = new_path / "docs"
    new_config_docs_path.mkdir()
    en_index_path: Path = en_docs_path / "docs" / "index.md"
    new_index_path: Path = new_config_docs_path / "index.md"
    en_index_content = en_index_path.read_text(encoding='utf-8')
    new_index_content = f"{missing_translation_snippet}\n\n{en_index_content}"
    new_index_path.write_text(new_index_content, encoding='utf-8')
    typer.secho(f"Successfully initialized: {new_path}", color=typer.colors.GREEN)
    update_languages(lang=None)
```
2. Result:
```bash
python ./scripts/docs.py new-lang zh
Successfully initialized: docs\zh
Updating en
Updating es
Updating zh
```

Should I create a PR for this?
### Environment

- OS: [Windows]
- FastAPI Version [0.52.0], get it with:

```bash
python -c "import fastapi; print(fastapi.__version__)"
```

- Python version 3.8.0, get it with:

```bash
python --version
```

**Comment by waynerv** [comment]
This problem has been already fixed in recent commit 459f0e11e5e09b7a380f3f859b58e20455378f50 ,so I will close this issue.

**Comment by tiangolo** [comment]
Thanks for reporting back and closing the issue @waynerv :+1:
